src/main/resources/data/minecolonies/tags/items/glassblower_ingredient.json

```diff
@@ -1,7 +1,8 @@
-{
-  "replace": false,
-  "values": [
-    "#forge:glass",
-    "#forge:glass_panes"
-  ]
-}
+{
+  "replace": false,
+  "values": [
+    "#forge:glass",
+    "#forge:glass_panes",
+    "minecraft:glass_bottle"
+  ]

```

Comment:
What is the reason for this?

Reply:
Figured it was a harmless way to add some mod compatibility for mods which use glass bottles as recipes. If you feel this is outside the scope of what should be included in this PR I can remove it. I'm more concerned about the gem issue (diamonds specifically) anyways

Reply:
I feel it could allow for all kind of recipes like filling up bottles with sth.

Reply:
In theory yes, it could lead to undesirable recipes for the glassblower. Maybe it should be handled on a case-by-case basis with other mods for compatibility. I can scrap this change if that's more aligned with how you feel crafting should work.

Reply:
yeah, better scrap this

Reply:
done, reverted